.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+node_modules
```

Comment:
- [ ] Kindly make sure your project has a file named `test.md` with the text "This file should be ignored by git" and which ignored by **Git** i.e the project name, ``test.md``, should be added inside the `.gitignore` file